environments/lcls/fel/observables.py

```diff
@@ -0,0 +1,83 @@
+
+import logging
+import time
+from badger.stats import percent_80
+from badger.errors import BadgerEnvObsError
+import numpy as np
+
+
+def get_intensity_n_loss(hxr, points, loss_pv, fel_channel, interface):
```

Comment:
lets add some documentation to these functions, including a docstring that explains what each variable is (see example https://numpydoc.readthedocs.io/en/latest/format.html) and type hints for each function argument